extra/3-magic-8-ball.js

```diff
@@ -47,6 +47,24 @@
 // and return the answer.
 function shakeBall() {
   //Write your code in here
+
+  let veryPositiveAnswers = ["It is certain.","It is decidedly so.", "Without a doubt.", "Yes - definitely.", "You may rely on it."];
+  let positiveAnswers = ["As I see it, yes.", "Most likely.", "Outlook good.", "Yes.", "Signs point to yes."];
+  let negativeAnswers = ["Reply hazy, try again.", "Ask again later.", "Better not tell you now.", "Cannot predict now.", "Concentrate and ask again."];
+  let veryNegativeAnswers = ["Don't count on it.", "My reply is no.", "My sources say no.", "Outlook not so good.", "Very doubtful."];
+
+  let randomNumber = Math.floor(Math.random() * veryPositiveAnswers.length);
+ 
+    let randomAnswer = [
+      veryPositiveAnswers[randomNumber],
+      positiveAnswers[randomNumber],
+      negativeAnswers[randomNumber],
+      veryNegativeAnswers[randomNumber],
+    ];
+
+  let randomNumber2 = Math.floor(Math.random() * randomAnswer.length);
+  console.log("The ball has shaken!");
+  return randomAnswer[randomNumber2];
```

Comment:
Well done Selahaddin. I like seeing your solutions, as it was a slighlty different apporach to what I had done. Good coding, keep it up!

Reply:
Thanks, Pete. It is good to hear.